origins_workflow/config/optional/workflows.workflow.nics_editorial_workflow.yml

```diff
@@ -1,17 +1,20 @@
+uuid: c0e58424-80ea-418b-bfd5-55ed7fce871a
```

Comment:
Do we want a `uuid` value in here? I think they're normally unique for a site and kept in the items flushed to the config/sync directory. This might not work for new Unity sites (haven't tested/confirmed) but I think it's safe to remove if this is starter config for the module.

Reply:
good point, that's done